### PR TITLE
Add iOS implementation

### PR DESCRIPTION
**Pull request by densa**

### CODE REVIEW

File: android/src/main/java/com/tipsi/cameraroll/RNTipsiCamerarollModule.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    }
+
+    @Override
+    public String getName() {
+        return "CameraRoll";
+    }
+
+    @ReactMethod
+    public void saveToCameraRoll(final String uri, String title, final Promise promise) {
+        promise.reject("Not implented yest");
```
Comment (sergeyzhukov) [inline android/src/main/java/com/tipsi/cameraroll/RNTipsiCamerarollModule.java]: typo. imple**me**nted yet

File: example/ios/example-tvOS/Info.plist [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+	</array>
+	<key>UISupportedInterfaceOrientations</key>
+	<array>
+		<string>UIInterfaceOrientationPortrait</string>
+		<string>UIInterfaceOrientationLandscapeLeft</string>
+		<string>UIInterfaceOrientationLandscapeRight</string>
+	</array>
+	<key>UIViewControllerBasedStatusBarAppearance</key>
+	<false/>
+	<key>NSLocationWhenInUseUsageDescription</key>
```
Comment (sergeyzhukov) [inline example/ios/example-tvOS/Info.plist]: please also add NSPhotoLibraryUsageDescription and NSPhotoLibraryAddUsageDescription to plist.info

File: ios/RNTipsiCameraroll/RNTipsiCameraroll.m [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    __block PHObjectPlaceholder *placeholder;
+    [[PHPhotoLibrary sharedPhotoLibrary] performChanges:^{
+
+        PHAssetChangeRequest *assetRequest = [PHAssetChangeRequest creationRequestForAssetFromImage:image];
+        placeholder = [assetRequest placeholderForCreatedAsset];
+
+        if(album) {
+            PHAssetCollectionChangeRequest *albumChangeRequest = [PHAssetCollectionChangeRequest changeRequestForAssetCollection:album];
+            [albumChangeRequest addAssets:@[placeholder]];
+
```
Comment (sergeyzhukov) [inline ios/RNTipsiCameraroll/RNTipsiCameraroll.m]: remove empty line

File: README.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
+
```
Comment (isnifer) [inline README.md]: remove empty line

File: android/build.gradle [hunk 5]
```diff
@@ -0,0 +1,35 @@
+
```
Comment (isnifer) [inline android/build.gradle]: remove empty line

File: android/src/main/AndroidManifest.xml [hunk 6]
```diff
@@ -0,0 +1,5 @@
+
```
Comment (isnifer) [inline android/src/main/AndroidManifest.xml]: remove empty line

File: android/src/main/java/com/tipsi/cameraroll/RNTipsiCamerarollModule.java [hunk 7]
```diff
@@ -0,0 +1,55 @@
+
```
Comment (isnifer) [inline android/src/main/java/com/tipsi/cameraroll/RNTipsiCamerarollModule.java]: remove empty line

File: android/src/main/java/com/tipsi/cameraroll/RNTipsiCamerarollPackage.java [hunk 8]
```diff
@@ -0,0 +1,28 @@
+
```
Comment (isnifer) [inline android/src/main/java/com/tipsi/cameraroll/RNTipsiCamerarollPackage.java]: remove empty line

File: example/.buckconfig [hunk 9]
```diff
@@ -0,0 +1,6 @@
+
```
Comment (isnifer) [inline example/.buckconfig]: remove empty line

File: example/__tests__/01_save_image.test.js [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+	await successSave(t, driver)
+
+
+	await driver.waitForVisible(saveBase64Id, 30000)
+	t.pass('User should see `Save from base64` button')
+
+	await driver.click(saveBase64Id)
+	t.pass('User should be able click on `Save from base64` button')
+	await successSave(t, driver)
+})
```
Comment (isnifer) [inline example/__tests__/01_save_image.test.js]: add empty line after

File: example/__tests__/utils/tape.js [hunk 11]
```diff
@@ -0,0 +1,15 @@
...
+    try {
+      await callback(t)
+    } catch (error) {
+      await helper.source()
+      await helper.screenshot()
+
+      throw error
+    }
+  })
+}
```
Comment (isnifer) [inline example/__tests__/utils/tape.js]: add empty line after

File: example/utils/testID.js [hunk 12]
```diff
@@ -0,0 +1,7 @@
+import { Platform } from 'react-native'
+
+export default function testID(id) {
+  return Platform.OS === 'android' ?
+    { accessible: true, accessibilityLabel: id } :
+    { testID: id }
+}
```
Comment (isnifer) [inline example/utils/testID.js]: add empty line after

File: index.js [hunk 13]
```diff
@@ -0,0 +1,6 @@
+
```
Comment (isnifer) [inline index.js]: remove empty line

File: ios/RNTipsiCameraroll/RNTipsiCameraroll.m [hunk 14]
```diff
@@ -0,0 +1,98 @@
...
```
Comment (isnifer) [inline ios/RNTipsiCameraroll/RNTipsiCameraroll.m]: remove empty line

**Comment by isnifer** [comment]
image
Also, these sections need to be fixed

**Review by isnifer — APPROVED** [review]
LGTM

**Pull request merged**